tests/unit/nutriscore_2023.t

```diff
@@ -0,0 +1,464 @@
+#!/usr/bin/perl -w
+
+use Modern::Perl '2017';
+use utf8;
+
+use JSON;
+
+use Test::More;
+use Test::Number::Delta relative => 1.001;
+use Log::Any::Adapter 'TAP';
+
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Tags qw/:all/;
+use ProductOpener::Food qw/:all/;
+use ProductOpener::Ingredients qw/:all/;
+use ProductOpener::Nutriscore qw/:all/;
+use ProductOpener::NutritionCiqual qw/:all/;
+use ProductOpener::NutritionEstimation qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+use Data::DeepAccess qw(deep_exists);
+
+my ($test_id, $test_dir, $expected_result_dir, $update_expected_results) = (init_expected_results(__FILE__));
+
+# Needed to compute estimated nutrients
+load_ciqual_data();
+
+my @tests = (
```

Comment:
Could we have those tests independently checked by some one of EREN, or someone reading the spec ?